config/tomee.yml

```diff
@@ -0,0 +1,57 @@
+# Cloud Foundry Java Buildpack
+# Copyright 2013-2015 the original author or authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Configuration for the Tomcat container
+---
+tomee:
+#  version: 8.0.+
+  version: 1.7.+
+  repository_root: ! 'https://tomee-repo.cfapps.pez.pivotal.io/tomee'
```

Comment:
This won't be the final home and shouldn't be committed to `master`.  Go ahead and put the `'{default.repository.root}'` version in place and we'll get TomEE dependencies uploaded to the global repository.  Any idea how far back in the TomEE line this will support?